packages/mobile/src/account/LinkBankAccountScreen.test.tsx

```diff
@@ -0,0 +1,282 @@
+import PersonaButton from './Persona'
+import Button from '@celo/react-components/components/Button'
+import openPlaid from 'src/account/openPlaid'
+import { createMockStore } from 'test/utils'
+import { mockAccount, mockPrivateDEK } from 'test/values'
+import { Provider } from 'react-redux'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { CICOEvents } from 'src/analytics/Events'
+import { fetchFinclusiveKyc } from './actions'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+
+let personaButtonSuccessCallback: (() => any) | undefined // using this to simulate Persona success at any arbitrary time
+let personaButtonErrorCallback: (() => any) | undefined // using this to simulate Persona error at any arbitrary time
+let personaButtonCancelCallback: (() => any) | undefined // using this to simulate Persona cancel at any arbitrary time
+
+const MockPersona = ({
+  onSuccess,
+  onPress,
+  onError,
+  onCancelled,
+}: {
+  onCancelled: () => any
+  onError: () => any
+  onSuccess: () => any
+  onPress: () => any
+}) => {
+  personaButtonSuccessCallback = onSuccess
+  personaButtonErrorCallback = onError
+  personaButtonCancelCallback = onCancelled
+  return <Button onPress={onPress} text="test persona button" testID="PersonaButton" />
+}
+
+const MOCK_PHONE_NUMBER = '+18487623478'
+
+jest.mock('src/analytics/ValoraAnalytics')
+
+jest.mock('./Persona', () => ({
+  __esModule: true,
+  namedExport: jest.fn(),
+  default: jest.fn(),
+}))
+
+jest.mock('src/account/openPlaid', () => ({
+  __esModule: true,
+  namedExport: jest.fn(),
+  default: jest.fn(),
+}))
+
+describe('LinkBankAccountScreen: unit tests (test one component at a time)', () => {
+  beforeEach(() => {
+    jest.useRealTimers()
+    jest.clearAllMocks()
+  })
+
+  describe('Tests with mock Persona button and openPlaid', () => {
+    beforeAll(() => {
+      //@ts-ignore . my IDE complains about this, though jest allows it
+      PersonaButton.mockImplementation(MockPersona)
+    })
+    afterEach(() => {
+      personaButtonSuccessCallback = undefined
+    })
+    afterAll(() => {
+      jest.clearAllMocks()
+    })
+    describe('StepOne', () => {
+      it('periodically polls ihl if KycStatus is approved but FinclusiveKycStatus is not', async () => {
+        const store = createMockStore({
+          account: {
+            kycStatus: KycStatus.Approved,
+            finclusiveKycStatus: FinclusiveKycStatus.InReview,
+          },
+        })
+        store.dispatch = jest.fn()
+        const { getByText } = render(
+          <Provider store={store}>
+            <StepOne />
+          </Provider>
+        )
+        await waitFor(() => getByText('linkBankAccountScreen.pending.title'))
+        expect(store.dispatch).toHaveBeenCalledWith(fetchFinclusiveKyc())
+        await waitFor(() => expect(store.dispatch).toHaveBeenCalledTimes(2), { timeout: 6000 })
+      }, 7000)
```

Comment:
that's a long test! can you use a fake timer instead? https://jestjs.io/docs/timer-mocks#run-all-timers